### PR TITLE
Issue 6359 - Add exception to GPL license in license tag

### LINKED ISSUE #6359: Add exception to GPL license in license tag
**Issue Description**
Add exception to GPL license in license tag

See https://gitlab.com/fedora/legal/fedora-license-data/-/issues/517

### PR DESCRIPTION
**Pull request by vashirov**
Description:
Update license string generator tool with the GPL exception. See https://gitlab.com/fedora/legal/fedora-license-data/-/issues/517

Fixes: https://github.com/389ds/389-ds-base/issues/6359

Reviewed by:

### CODE REVIEW

File: rpm/389-ds-base.spec.in [hunk 1]
```diff
@@ -67,7 +67,7 @@ Summary:          389 Directory Server (%{variant})
 Name:             389-ds-base
 Version:          __VERSION__%{?prerel}
 Release:          %{autorelease -n %{?with_asan:-e asan}}%{?dist}
-License:          GPLv3+ and (ASL 2.0 or MIT)
+License:          GPL-3.0-or-later WITH GPL-3.0-389-ds-base-exception AND (0BSD OR Apache-2.0 OR MIT) AND (Apache-2.0 OR Apache-2.0 WITH LLVM-exception OR MIT) AND (Apache-2.0 OR BSL-1.0) AND (Apache-2.0 OR MIT OR Zlib) AND (Apache-2.0 OR MIT) AND (CC-BY-4.0 AND MIT) AND (MIT OR Apache-2.0) AND Unicode-DFS-2016 AND (MIT OR CC0-1.0) AND (MIT OR Unlicense) AND 0BSD AND Apache-2.0 AND BSD-2-Clause AND BSD-3-Clause AND ISC AND MIT AND MIT AND ISC AND MPL-2.0 AND PSF-2.0
```
Comment (tbordaz) [inline rpm/389-ds-base.spec.in:70]: Here we are hardcoding the licence. Does it mean that this change obsoletes the tool generating the license string ?

**Comment by progier389** [comment]
Should we explain somewhere (Maybe in LiCENSE file?) what GPL-3.0-389-ds-base-exception exactly means 
(i.e referring to the LICENSE.openssl file) ?
Another point: The LICENSE file does not list the LICENSE.openldap file. Is it normal ?

File: rpm/389-ds-base.spec.in (hunk 1 above)
Comment (vashirov) [reply rpm/389-ds-base.spec.in:70]: This string will get replaced after running the script anyway. I didn't want to put incorrect license string. Maybe we should replace it with a placeholder saying to run the script?

**Comment by vashirov** [comment]
Sorry, I can't answer these questions, I'm just a messenger :) Let's ask our legal team.

**Review by droideck — APPROVED** [review]
Looks good to me!
IMO, we can improve the license mentions in another PR. This PR is self-sufficient enough, in my opinion.

**Pull request merged**